cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -69,9 +70,8 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         }
         note {
           p(
-            """When creating a user, their initial list of roles is empty.
-              |In Neo4j community-edition this is not important as all users have implied administator privileges.
-              |In Neo4j enterprise-edition a new user like this will have no rights and it is important to <<administration-security-roles-grant, grant roles to the user>>.""".stripMargin)
+            """In Neo4j community-edition there are no roles, but all users have implied administrator privileges.
+              |In Neo4j enterprise-edition all users are automatically assigned the <<administration-security-roles-public, `PUBLIC`>> role, giving them a base set of privileges.""".stripMargin)
```

Comment:
Please could we mirror the styling on [this page ](https://neo4j.com/docs/operations-manual/current/introduction/#editions) when we mention the editions of Neo4j:

**Enterprise Edition** and **Community Edition**